vyper/semantics/analysis/module.py

```diff
@@ -331,10 +331,7 @@ def validate_initialized_modules(self):
         # don't call `__init__()` for modules which don't have
         # `__init__()` function
         for m in should_initialize.copy():
-            for f in m.functions.values():
-                if f.is_constructor:
-                    break
-            else:
+            if not any(f.is_constructor for f in m.functions.values()):
```

Comment:
why do we use both `is_constructor` and few lines below `dep_t.init_function`? can we unify?

Reply:
good idea, see 132b4876bd045bead1d97c46da54cd35948e757f

Reply:
349a39bed4d2913e692bcec75ee1ef6aa2fa851e as well